src/helix-ui/styles/elements/hx-tile.less

```diff
@@ -1,5 +1,6 @@
 hx-tile {
-  &:extend(.hxBox-lg);
+  &:extend(.hxBox-lg); // DEPRECATED: remove in v1.0.0
+  &:extend(.hxBox.hxLg);
```

Comment:
Please use `#Box.md()` (`1.25rem`) to match previous `.hxBox-lg` (`1.25rem`) padding.